src/scripts/main.js

```diff
@@ -1,8 +1,41 @@
 'use strict';
 
 const people = require('./lib/people');
+const getTable = document.querySelector('.dashboard');
```

Comment:
```suggestion
const table = document.querySelector('.dashboard');
```
Это не функция, в этой переменной хранится сама таблица